src/style.css

```diff
@@ -2,0 +1,90 @@
+body {
+  font-family: ' Avenir', Arial, sans-serif;
+}
+
+input {
+  all: unset;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+}
+
+@font-face {
+  font-family: ' Avenir', sans-serif;
```

Comment:
The 'sans-serif' fallback font should not be included in the @font-face rule since it's defining a custom font, not a font stack. The fallback should be specified where the font-family is used, like in the body rule.